src/environments.js

```diff
@@ -66,10 +66,12 @@ var environmentDefinitions = [
             var parser = this;
             // Currently only supports alignment, no separators like | yet.
             colalign = colalign.value.map ? colalign.value : [colalign];
-            colalign = colalign.map(function(node) {
+            var cols = colalign.map(function(node) {
                 var ca = node.value;
                 if ("lcr".indexOf(ca) !== -1) {
-                    return ca;
+                    return {
+                        align: ca
+                    };
```

Comment:
I wouldn't worry about it right now.  Whenever we tackle the "format description" part of https://github.com/Khan/KaTeX/issues/269 would be a good time to refactor this.  I like ```width``` instead of ```value``` b/c it conveys more meaning.  Is the ```index``` field necessary for content types?